azure-pipelines.yml

```
@@ -392,9 +392,13 @@ jobs:
       verilator --version
     displayName: Display environment
   - bash: |
-      export PATH="/opt/buildcache/riscv/bin:$PATH"
+      export PATH="${TOOLCHAIN_PATH}/bin:$PATH"
```

```suggestion
      export PATH="$TOOLCHAIN_PATH/bin:$PATH"
```

(same below) for consistency within this file.

Or even better: use 

```
echo "##vso[task.prependpath]$TOOLCHAIN_PATH/bin"
```
in the "Install toolchain" step to have PATH adjusted for all subsequent jobs.

(https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#prependpath-prepend-a-path-to-the--path-environment-variable)

---

Ah, neat, I'll do that. Thanks!